src/Umbraco.Core/Persistence/PetaPocoSqlExtensions.cs

```diff
@@ -34,11 +34,27 @@ public static Sql Where<T>(this Sql sql, Expression<Func<T, bool>> predicate)
             return sql.Where(whereExpression, expresionist.GetSqlParameters());
         }
 
+        private static string GetFieldName<T>(Expression<Func<T, object>> fieldSelector, ISqlSyntaxProvider sqlSyntax)

```

Comment:
The name of this method bit confusing as it returns the fieldName & the tableName, as I was confused at first glance that why OrderBy method was now just two lines